tools/create-genesis/src/contracts.rs

```diff
@@ -0,0 +1,721 @@
+use crate::common::clean_0x;
```

Comment:
What's used for since we already have init_data?

Reply:
We need to get contract info list from `contracts.yml`.